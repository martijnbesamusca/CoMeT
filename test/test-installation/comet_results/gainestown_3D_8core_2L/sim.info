{'cmdline': ['/home/martijn/Documents/uva/thesis/CoMeT/run-sniper',
             '-v',
             '-s',
             'memTherm_core',
             '-n',
             '4',
             '-c',
             'gainestown_3D_8core_2L',
             '--',
             '/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/test'],
 'git_revision': '70a94ea0a46a09195fc7cc97e7411ec90733f6a3',
 'host': '114c1c70b953',
 'pin_version': 'unknown',
 'rusage': (22.656988,
            1.5232160000000001,
            108812,
            0,
            0,
            0,
            50823,
            0,
            0,
            0,
            3424,
            0,
            0,
            0,
            260262,
            341),
 'snipercmd': ['bash',
               '-c',
               '/home/martijn/Documents/uva/thesis/CoMeT/lib/sniper -c /home/martijn/Documents/uva/thesis/CoMeT/config/base.cfg --general/total_cores=4 --general/output_dir=/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation --config=/home/martijn/Documents/uva/thesis/CoMeT/config/nehalem.cfg --config=/home/martijn/Documents/uva/thesis/CoMeT/config/gainestown_3D_8core_2L.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/sim.scripts.py -g --hooks/script0args= -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpv0RU4i/run_benchmarks'],
 't_elapsed': 27.382986068725586,
 't_start': 1653910888.42506,
 'tracecmds': [['/home/martijn/Documents/uva/thesis/CoMeT/record-trace',
                '-o',
                '/tmp/tmpv0RU4i/run_benchmarks',
                '-v',
                '--pa',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                '/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/test']],
 'user': None,
 'vmem': 791568384L}
