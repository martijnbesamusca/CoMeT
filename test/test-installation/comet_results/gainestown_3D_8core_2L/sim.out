                                     | Core 0     | Core 1     | Core 2     | Core 3     | Core 4     | Core 5     | Core 6     | Core 7    
  Instructions                       |   21246362 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  Cycles                             |    8831520 |    8831520 |    8831520 |    8831520 |    8831520 |    8831520 |    8831520 |    8831520
  IPC                                |       2.41 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
  Time (ns)                          |    2453200 |    2453200 |    2453200 |    2453200 |    2453200 |    2453200 |    2453200 |    2453200
  Idle time (ns)                     |        -90 |    2453200 |    2453200 |    2453200 |    2453200 |    2453200 |    2453200 |    2453200
  Idle time (%)                      |      -0.0% |     100.0% |     100.0% |     100.0% |     100.0% |     100.0% |     100.0% |     100.0%
Branch predictor stats               |            |            |            |            |            |            |            |           
  num correct                        |    2597555 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  num incorrect                      |      26946 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  misprediction rate                 |      1.03% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
  mpki                               |       1.27 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
TLB Summary                          |            |            |            |            |            |            |            |           
  I-TLB                              |            |            |            |            |            |            |            |           
    num accesses                     |    2890054 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    num misses                       |         46 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    miss rate                        |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
  D-TLB                              |            |            |            |            |            |            |            |           
    num accesses                     |    7876038 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    num misses                       |        905 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    miss rate                        |      0.01% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.04 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
  L2 TLB                             |            |            |            |            |            |            |            |           
    num accesses                     |        951 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    num misses                       |        422 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    miss rate                        |     44.37% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.02 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
Cache Summary                        |            |            |            |            |            |            |            |           
  Cache L1-I                         |            |            |            |            |            |            |            |           
    num cache accesses               |    2890054 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    num cache misses                 |        924 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    miss rate                        |      0.03% |       inf% |       inf% |       inf% |       inf% |       inf% |       inf% |       inf%
    mpki                             |       0.04 |        inf |        inf |        inf |        inf |        inf |        inf |        inf
  Cache L1-D                         |            |            |            |            |            |            |            |           
    num cache accesses               |    7876038 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    num cache misses                 |      49509 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    miss rate                        |      0.63% |       inf% |       inf% |       inf% |       inf% |       inf% |       inf% |       inf%
    mpki                             |       2.33 |        inf |        inf |        inf |        inf |        inf |        inf |        inf
  Cache L2                           |            |            |            |            |            |            |            |           
    num cache accesses               |      50917 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    num cache misses                 |      49001 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    miss rate                        |     96.24% |       inf% |       inf% |       inf% |       inf% |       inf% |       inf% |       inf%
    mpki                             |       2.31 |        inf |        inf |        inf |        inf |        inf |        inf |        inf
DRAM summary                         |            |            |            |            |            |            |            |           
  num dram accesses                  |      10152 |      10168 |      10192 |      10172 |      10211 |      10619 |      10126 |      10153
  average dram access latency (ns)   |      23.47 |      23.47 |      23.48 |      23.46 |      23.52 |      34.42 |      23.45 |      23.46
  average dram queueing delay        |       0.04 |       0.04 |       0.05 |       0.03 |       0.09 |      10.99 |       0.02 |       0.03
  average dram bandwidth utilization |      3.00% |      3.00% |      3.00% |      3.00% |      3.00% |      3.00% |      3.00% |      3.00%
Coherency Traffic                    |            |            |            |            |            |            |            |           
  num loads from dram                |      16334 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  num loads from dram cache          |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  num loads from remote cache        |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0
